src/tokenizer/mod.rs

```diff
@@ -651,6 +652,11 @@ impl<Sink: TokenSink> Tokenizer<Sink> {
 
         h5e_debug!("processing in state {:?}", self.state);
         match self.state {
+            states::Quiescent => {
+                self.state = states::Data;
+                return false;
+            }
```

Comment:
This could use a comment along the lines of

> Reachable only through `query_state_change`. The tree builder wants the tokenizer to suspend processing.